frontend/src/scenes/trends/viz/ActionsLineGraph.tsx

```diff
@@ -39,7 +39,7 @@ export function ActionsLineGraph({
             showPersonsModal={showPersonsModal}
             tooltipPreferAltTitle={filters.insight === ViewType.STICKINESS}
             onClick={
-                dashboardItemId
+                dashboardItemId || filters.formula
```

Comment:
It's not obvious to me why these cases should mean that we don't have `onClick`. Can we add either a comment or provide as a function `isPersonsListValidForFilter`.

Reply:
good point